spec/features/issue_trash_recover_spec.rb

```diff
@@ -15,6 +15,8 @@
     visit "/trash"
     click_link 'Recover'
 
+    sleep(1)
```

Comment:
What's the source cause of this behavior? Is there a way around introducing a 1 second delay in the process? 
/cc @georgemillo @aapomm @dormi